proposals/0035-linalg-matrix.md

```diff
@@ -833,6 +917,28 @@ Populates a matrix with data from a `groupshared` array. Data conversions
 between opaque matrices and groupshared memory are defined in the
 [Conversions](#conversions) section below.
 
+```llvm
+declare void @dx.op.matrixLoadFromThreads.v[NUM][TY](
+  immarg i32,            ; opcode
+  %dx.types.MatrixRef *, ; matrix
+  < NUM x [Ty]>,         ; Vector
+  )
+```
+
+Populates a matrix from per-thread vectors. For an A matrix the NUM corresponds
+to the M dimension while for a B matrix it corresponds to the N dimension. The
+NUM must match the matrix corresponding dimension, unless the element is a
+packed data type in which case it must be the number of 32-bit unsigned integers
+used to store M elements. This operation may not be used on Accumulator
+matrices.
+
+For an A matrix the N dimension must be less than or equal to the WaveSize. For
+a B matrix the M dimension must be less than or equal to the WaveSize. Values
+from additional threads are discarded.

```

Comment:
Don't you mean the `K` dimension for both A and B matrix case?
```suggestion
For an A matrix the K dimension must be less than or equal to the WaveSize. For
a B matrix the K dimension must be less than or equal to the WaveSize. Values
from additional threads are discarded.
```

Reply:
This proposal doesn't consider `K` dimensions the way that the old WaveMatrix one did. All matrices are `M x N`, the `K` dimension only factors into the multiplication operations where the matrices are `M x K` and `K x N` to produce an `M x N` output.